gateway/src/apicast/policy_schemas_loader.lua

```diff
@@ -0,0 +1,99 @@
+--- Policy schemas loader
+-- Finds the schemas for the builtin policies and the other ones loaded.
+
+local pl_file = require('pl.file')
+local pl_dir = require('pl.dir')
+local pl_path = require('pl.path')
+local cjson = require('cjson')
+local format = string.format
+local ipairs = ipairs
+local insert = table.insert
+local policy_loader = require('apicast.policy_loader')
+
+local _M = {}
+
+local builtin_policy_load_path = policy_loader.builtin_policy_load_path
+local policy_load_paths = policy_loader.policy_load_paths
+
+local policy_schema_name = 'apicast-policy.json'
+
+local function dir_iter(dir)
+  return ipairs(pl_dir.getdirectories(dir))
+end
+
+-- Returns the schemas for all the built-in policies.
+local function all_builtin_policy_schemas()
+  local schemas = {}
+
+  for _, policy_dir in dir_iter(builtin_policy_load_path()) do
+    local schema_file = format('%s/%s', policy_dir, policy_schema_name)
+    local schema = pl_file.read(schema_file)
```

Comment:
Just a nitpick. It is not a schema, but a manifest, right?

Reply:
:+1: